docs/docs/configuration/record.md

```diff
@@ -161,6 +161,25 @@ Using Frigate UI, HomeAssistant, or MQTT, cameras can be automated to only recor
 
 The export page in the Frigate WebUI allows for exporting real time clips with a designated start and stop time as well as exporting a time-lapse for a designated start and stop time. These exports can take a while so it is important to leave the file until it is no longer in progress.
 
+### Time-lapse export
+
+When exporting a time-lapse the default speed-up is 25x with 30 FPS. This means that every 25 seconds of (real-time) recording is condensed into 1 second of time-lapse video (always without audio) with a smoothness of 30 FPS.
+To configure the speed-up factor, the frame rate and further custom settings, the configuration parameter `timelapse_args` can be used. The below configuration example would change the time-lapse speed to 60x (for fitting 1 hour of recording into 1 minute of time-lapse) with 25 FPS:
+
+```yaml
+record:
+  enabled: True
+  export:
+    timelapse_args: "-vf setpts=PTS/60 -r 25"
+```
+
+:::warning
+
+When using `ffmpeg.hwaccel_args: preset-vaapi` globally, the resulting file size is typically undesirably large.
+To get a reasonable file size nevertheless, the ffmpeg parameter `-qp n` can be utilized (where `n` stands for the value of the quantisation parameter). Values of around 30 have shown to provide a reasonable tradeoff between quality and file size.
+
+:::
```

Comment:
I don't think we want to be specific here, this can occur with any hardware encoder and depends what the underlying levels of the hardware encoder choose.

Reply:
What about making the current "warning" less prominent?
Like changing it to a "tip" or just normal text. Also moving it to some kind of troubleshooting section would be an option...